Misc/NEWS.d/next/Documentation/2017-12-24-17-29-37.bpo-32360.eZe-ID.rst

```diff
@@ -0,0 +1 @@
+Remove references of OrderedDict in module `unittest`
```

Comment:
I find «reference» a bit vague, what do you think about using «import» instead?

Reply:
I don't think NEWS entry is required for this.
You can use "skip news" label instead